tests/test_factory.py

```diff
@@ -2,14 +2,18 @@
 import time
 from datetime import date, datetime
 
-import dateparser
 import pytest
 from dateutil import tz
 
 from arrow.parser import ParserError
 
 from .utils import assert_datetime_equality
 
+try:
+    import dateparser
+except ImportError:
+    dateparser = None
+
```

Comment:
Thanks for bringing this up. This is a great use case for pytest importorskip: https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency. Mind replacing this implementation with importorskip and we can get this merged in?

Reply:
Thanks, I didn't know about that (or expect such a thing to exist ;-)).

Reply:
Hah me neither! I just had a hunch that something like this existed so I did some googling and came across it!